Makefile

```diff
@@ -0,0 +1,41 @@
+# These targets are not files
+.PHONY: lint lint-scripts check-format format run publish
+
+STACK ?= heroku-24
+FIXTURE ?= test/spec/fixtures/repos/java-overlay-test
+
+# Converts a stack name of `heroku-NN` to its build Docker image tag of `heroku/heroku:NN-build`.
+STACK_IMAGE_TAG := heroku/$(subst -,:,$(STACK))-build
+
+lint: lint-scripts check-format
+
+lint-scripts:
+	@git ls-files -z --cached --others --exclude-standard 'bin/*' 'etc/*' 'lib/*' 'opt/*' | xargs -0 shellcheck --check-sourced --color=always
+
+check-format:
+	@shfmt -f . | grep -v "vendor/" | grep -v "test/spec/fixtures/" | xargs shfmt -i 2 --diff
+
+format:
+	@shfmt -f . | grep -v "vendor/" | grep -v "test/spec/fixtures/" | xargs shfmt -i 2 --write --list
+
+run:
```

Comment:
(optional) I don't know if you saw in the Python repo, but I also run this target in CI to (a) check the development workflow works, (b) enforce in CI that `bin/report` exits successfully (given it can't easily be tested in Hatchet tests, since errors are ignored there):
https://github.com/heroku/heroku-buildpack-python/blob/b90a4755ae2008a5cdae2b32632a9fb4da49db0d/.github/workflows/ci.yml#L66-L76

Reply:
Thanks! Yes, I saw that but forgot to port it over too. I now did so (without the failing case for now) in [0c6e11d](https://github.com/heroku/heroku-buildpack-jvm-common/pull/318/commits/0c6e11d5246228ff123ab20800d98cc971a94ab2).